docs/markdown/snippets/test-verbose.md

```diff
@@ -0,0 +1,8 @@
+## New keyword argument `verbose` for tests and benchmarks
+
+The new keyword argument `verbose` can be used to mark tests and benchmarks
+that must always be logged verbosely on the console.  This is particularly
+useful for long-running tests, or when a single Meson test() is wrapping
+an external test harness.  In this case, TAP should be used by the external
+harness so that Meson will log each subtest as it runs, instead of including
+the log at the end of the run.
```

Comment:
The last sentence does not make sense to me. What is "in this case"? The thing it refers to does not even appear anywhere else in the snippet.

Reply:
It refers to "Meson test() is wrapping an external test harness".

Reply:
But why does it talk about TAP? This is not directly tied to TAP as far as I can see.

Reply:
Without TAP, the output is logged all at once at the end of the run. It is better if the external harness uses TAP, because then `meson test` will separately log each `ok`/`not ok` line that the external harness emits.

Reply:
This explanation should be in the snippet itself so people reading it can understand what is happening.

Reply:
I am not sure what was missing in the snippet, so I moved the whole explanation to the reference manual instead. This makes it possible to be less concise.